src/main/java/org/usfirst/frc/team2473/framework/CHS_SparkMax.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc.team2473.framework;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class CHS_SparkMax {
+	private CANSparkMax spark;
+	private CANEncoder encoder;
+
+	private double lastEncoderValue = 0;
+
+	public CHS_SparkMax(int port, MotorType motorType) {
+		spark = new CANSparkMax(port, motorType);
```

Comment:
Why are we using CANSparkMax over Spark (http://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/Spark.html)?

Reply:
Spark is the old Spark controller. The new Spark MAX controllers have a separate API ([http://www.revrobotics.com/content/sw/max/sw-docs/java/com/revrobotics/CANSparkMax.html](url))